format_implementing.md

```diff
@@ -16,9 +16,34 @@ Framework IDs SHOULD be prefixed with `framework-`. This naming scheme makes it
 
 When a user downloads a project using a launcher, it's important to download its required dependencies. Launchers should combine the version dependencies with the file dependencies in order to create a full dependency list.
 
-For the purposes of this section, the term "the host" refers to the service a launcher is using in order to download projects, such as Diluv or Modrinth.
+When looking at a dependency, the first thing to do is check for the `src` field. If the `src` field exists, then the mod has external MODIP metadata. Process the metadata for that dependency from the `src` field.
+
+If there is no `src` field, you have two options:
+1) Request information on this ID from hosts that your launcher knows of.
+2) Throw a warning to the user, suggesting that they need to download the project, but the launcher cannot automatically install it.
+
+
+### Reasoning for optional `src` field
+
+You might think having an optional `src` field is stupid, as it doesn't tell a launcher where to get a dependency. But there is a valid reason for it. Some dependencies cannot be represented in MODIP format at all, due to legal, technical, or other reasons. The optional `src` field allows developers to indicate that a dependency is needed, but due to any number of reasons it cannot be provided. 
+
+This is better than simply ommitting the dependency altogether, as it gives a launcher the opportunity to notify the user about the dependency, rather than missing it.
+
+### Example
+
+In the below example, a launcher should display a message indicating that `non-modip-dependency` cannot be automatically downloaded.
+```json
+"dependencies": [
+  {
+    "id": "non-modip-dependency",
+    "name": "Non-MODIP Dependency",
+    "version": "1.0.0",
+    // notice: no `src` field
```

Comment:
No mod ID or expected class files either?

Reply:
Those will be added in a later PR.